sdk/Cargo.toml

```diff
@@ -66,6 +68,9 @@ libsecp256k1 = { version = "0.3.5", optional = true }
 sha3 = { version = "0.9.1", optional = true }
 digest = { version = "0.9.0", optional = true }
 
+[target.'cfg(not(target_arch = "bpf"))'.dependencies]
+curve25519-dalek = { version = "2.1.0" }
```

Comment:
freebie: this fixes a compile error when using `Pubkey::create_program_address()` from a program when not targeting BPF